scrapy/utils/python.py

```diff
@@ -174,17 +174,24 @@ def new_method(self, *args, **kwargs):
         return cache[self]
     return new_method
 
+
 _BINARYCHARS = {six.b(chr(i)) for i in range(32)} - {b"\0", b"\t", b"\n", b"\r"}
 _BINARYCHARS |= {ord(ch) for ch in _BINARYCHARS}
 
-
+@deprecated("scrapy.utils.python.binary_is_text")
 def isbinarytext(text):
-    """Return True if the given text is considered binary, or False
-    otherwise, by looking for binary bytes at their chars
+    """ This function has been removed.
+    Please use scrapy.utils.python.binary_is_text """
```

Comment:
I wonder if we should note `binary_is_text` does the opposite of `isbinarytext` in a docstring. On one hand, this is easy to check in source code; on the other hand, function names look too similar, it is easy to make a mistake and think the function was renamed.